config/application.yml

```diff
@@ -10,6 +10,8 @@ defaults: &defaults
       - "florian+cfp-errors@eurucamp.org"
       - "alex+cfp-errors@eurucamp.org"
   conference:
+    coc_link:
+      "https://rubyberlin.github.io/code-of-conduct"
```

Comment:
Use /code-of-conduct**/** – otherwise GitHub pages will 301-bounce you to the trailing forward slash (and remove `https` while doing so :panda_face:)